### PR TITLE
ptpwebcam 1.2.6 (new cask)

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew install --cask {{cask_file}}` worked successfully.
- [x] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Only one problem now:
```
audit for ptpwebcam: failed
 - GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
Error: 1 problem in 1 cask detected
```
I would be happy to make an exception in this case @SMillerDev if you will be actually using this Cask, however -- it is the least I can do for all your help in my PR's to `homebrew-core`.

File: Casks/ptpwebcam.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  version "1.2.6"
+  sha256 "6d77dbb9d6052397f953b49e4a218cb809ec864a9cf3dfc83318f788c71e88c4"
+
+  url "https://github.com/dognotdog/ptpwebcam/releases/download/v#{version}/PTP_Webcam-v#{version}.pkg",
+      verified: "https://github.com/dognotdog/ptpwebcam"
+  name "ptpwebcam"
+  desc "DSLR live view video plugin"
+  homepage "https://ptpwebcam.org/"
+
+  livecheck do
```
Comment (SMillerDev) [inline Casks/ptpwebcam.rb:11]: AFAIK all of these are standard for github.

Comment (vitorgalvao) [reply Casks/ptpwebcam.rb:11]: Looks like you are correct. But in that case we don’t need the `appcast` either. We should change the `audit` to no longer require it.

Comment (vitorgalvao) [reply Casks/ptpwebcam.rb:11]: https://github.com/Homebrew/brew/pull/10344

**Comment by miccal** [comment]
Thanks @SMillerDev.

**Review by miccal — APPROVED** [review]

**Pull request merged**